de-lib/src/main/java/org/iplantc/de/client/services/impl/DEUserSupportServiceFacadeImpl.java

```diff
@@ -12,29 +13,25 @@
 import com.google.web.bindery.autobean.shared.Splittable;
 
 /**
- * Provides access to remote services for submitting user feedback.
+ * Created by sriram on 12/1/16.
  */
-@SuppressWarnings("nls")
-public class DEFeedbackServiceFacadeImpl implements DEFeedbackServiceFacade {
+public class DEUserSupportServiceFacadeImpl implements DEUserSupportServiceFacade {
 
-    private static String FEEDBACK_SERVICE_PATH = "feedback";
+    private static String SUPPORT_SERVICE_PATH = "support-email";
```

Comment:
I also think that would be better. Then if the path changes, we only have to change the config and not the code.